### PR TITLE
Feature/fixed value heatmap

### PR DESCRIPTION
**Pull request by pacs01**

### CODE REVIEW

File: src/utils/InputHandler.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+				break;
+			case "text":
+			case "number":
+				callback(name, event.currentTarget.value);
+				break;
+		}
+	}
+};
+
+export type allInputTypes = string | number | boolean;
```
Comment (eluchsinger) [inline src/utils/InputHandler.jsx]: Need to define the types before they are used.

File: src/utils/InputHandler.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+export const inputHandler = (callback: (string, allInputTypes) => void) => (
+	event: SyntheticEvent<HTMLInputElement>
+) => {
+	if (
+		event.currentTarget != null &&
+		event.currentTarget.name != null &&
+		(event.currentTarget.value != null || event.currentTarget.checked != null)
+	) {
+		const name = event.currentTarget.name;
+		switch (event.currentTarget.type) {
```
Comment (eluchsinger) [inline src/utils/InputHandler.jsx]: Default case?

File: src/components/heatmapModeForm/HeatmapModeForm.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+	onChange: (string, allInputTypes) => void
+};
+
+export default function HeatmapModeForm({
+	configObject,
+	onSubmit,
+	onChange
+}: Props) {
+	return (
+		<Form heading="Modi aktivieren" onSubmit={() => {}}>
```
Comment (eluchsinger) [inline src/components/heatmapModeForm/HeatmapModeForm.jsx]: Doesn't require a submit button. Make the `onSubmit` callback optional in the `<Form />` component and make the button invisible, if there is no callback available.

File: src/utils/InputHandler.jsx (hunk 2 above)
Comment (pacs01) [reply src/utils/InputHandler.jsx]: If no html input type matches, then the event is somehow invalid or with an unsupported type. In this case the callback is not called which is the desired behavior.

**Pull request merged**